src/timeline.js

```diff
@@ -42,19 +42,37 @@ export default class Timeline extends React.Component {
   static TIMELINE_MODES = Object.freeze({
     SELECT: 1,
     DRAG: 2,
-    RESIZE: 4,
+    RESIZE: 4
   });
 
   static propTypes = {
-    items: PropTypes.arrayOf(PropTypes.object).isRequired,
+    items: PropTypes.arrayOf(
+      PropTypes.shape({
+        key: PropTypes.number.isRequired,
+        row: PropTypes.number.isRequired,
```

Comment:
cf. above: we shouldn't impose these in "items". So to be removed. I'd keep "key". However, I don't know what kind of enforcement is applied via these propTypes. I think I saw somewhere that they will maybe be deprecated? Somehow in relation w/ TypeScript.

Reply:
Done